server/app/controllers/admin/AdminQuestionTranslationsController.java

```diff
@@ -120,49 +103,49 @@ public CompletionStage<Result> edit(Http.Request request, long questionId, Strin
    *     same {@link QuestionTranslationView} with error messages
    */
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
-  public CompletionStage<Result> update(Http.Request request, long questionId, String locale) {
+  public Result update(Http.Request request, String questionName, String locale) {
```

Comment:
💁‍♂️ made this synchronous to make the code easier to read since it's admin-facing